torch/distributions/geometric.py

```diff
@@ -75,7 +75,8 @@ def probs(self):
     def sample(self, sample_shape=torch.Size()):
         shape = self._extended_shape(sample_shape)
         with torch.no_grad():
-            u = self.probs.new(shape).uniform_(_finfo(self.probs).tiny, 1)
+            u = torch.rand(shape, dtype=self.probs.dtype, device=self.probs.device)
+            u = u.clamp(min=_finfo(self.probs).tiny)
```

Comment:
Can you please mark this place as a workaround? We'll add the factory functions you need later, and this will make fining all such places easier. Maybe let's prefix all those comments with `# [JIT WORKAROUND] ...`